### PR TITLE
protocal/derivation: specify the reason why fill 127 bytes at once

### PR DESCRIPTION
**Pull request by joohhnnn**
the reason why use 4 elements here confusing me for a long time, and specify the reason why fill 127 bytes at once is a good explanation for time saving

### CODE REVIEW

File: specs/protocol/derivation.md [hunk 1]
```diff
@@ -535,7 +535,7 @@ describe the encoding of this polynomial.
 The field elements are encoded as big-endian integers (`KZG_ENDIANNESS = big`).
 
 To save computational overhead, only `254` bits per field element are used for rollup data.
-
+To optimize data encoding, `254` bits are defined as equivalent to `31.5` bytes. To ensure efficient and complete utilization of data, four elements are employed, summing to a total of `31.5` bytes multiplied by `4`, which yields `127` bytes.
```
Comment (protolambda) [inline specs/protocol/derivation.md]: 254 bits is equivalent to 31.75 bytes, not 31.5 bytes. Also please change the wording to be less verbose.

Comment (joohhnnn) [reply specs/protocol/derivation.md]: LOL sorry for the stupid math problem. Shame on that, but good to know you got my point ;)

**Review by protolambda — APPROVED** [review]

**Comment by protolambda** [comment]
Issue with line-length in markdown lint check.

**Review by protolambda — APPROVED** [review]

File: specs/protocol/derivation.md [hunk 2]
```diff
@@ -536,6 +536,9 @@ The field elements are encoded as big-endian integers (`KZG_ENDIANNESS = big`).
 
 To save computational overhead, only `254` bits per field element are used for rollup data.
 
+For efficient data encoding, `254` bits (equivalent to `31.75` bytes) are utilized. 
+`4` elements combine to effectively use `127` bytes.
+
```
Comment (protolambda) [inline specs/protocol/derivation.md]:
```suggestion
For efficient data encoding, `254` bits (equivalent to `31.75` bytes) are utilized.
`4` elements combine to effectively use `127` bytes.

```

Removes trailing space

**Review by protolambda — APPROVED** [review]

**Pull request merged**